data/compare.yaml

```diff
@@ -347,13 +347,13 @@ AblyCompare:
           pusher: '*No.*'
         extra: 'Storing connection state means clients can resume from where they left off, providing a better quality of service as nothing is ever lost.<br/><br/>"Find out more about connection state recovery":https://support.ably.io/solution/articles/3000044639'
       uptimeguarantee:
-        description: '100% uptime guarantee<br/><br/>(Unique to Ably)'
+        description: 'Uptime guarantee<br/><br/>'
         compare:
-          ably: '*Yes.*<br/><br/>Ably uniquely offers a 100% uptime guarantee to all "Business and Enterprise customers":https://www.ably.io/pricing. We stand by our 100% uptime promise - if we are unable to meet that goal, "we offer refunds":https://support.ably.io/solution/articles/3000029531.'
-          pubnub: '<b>No.</br>'
+          ably: '*Yes.*<br/><br/>Ably offers varying levels of uptime guarantees depending on your needs. For all of our enterprise customers we provide a 99.999% uptime SLA. If we're unable to meet your SLA goal, "we offer refunds":https://support.ably.io/solution/articles/3000029531.'
+          pubnub: '<b>No. PubNub doesn't offer an SLA unless you a Pro customer.</br>'
```

Comment:
I think the `'` in `doesn't` will break the formatting as it's wrapped in `'`. Maybe just make it a newline with > to start the line, as some other examples in this page have, to avoid future issues.